utils/ratelimits.py

```diff
@@ -2,9 +2,20 @@
 
 from flask import request, make_response, jsonify
 
+import rethinkdb as r
+import json
+
+config = json.load(open('config.json'))
+
+RDB_ADDRESS = config['rdb_address']
+RDB_PORT = config['rdb_port']
+RDB_DB = config['rdb_db']
+
+rdb = r.connect(RDB_ADDRESS, RDB_PORT)
```

Comment:
`db=RDB_DB`, this removes the need to do `r.db(RDB_DB).table(...)` on each call.